dsl/camel-jbang/camel-jbang-it/src/test/java/org/apache/camel/dsl/jbang/it/support/JBangTestSupport.java

```diff
@@ -185,6 +185,26 @@ protected void checkLogContains(String route, String contains, int waitForSecond
                         .contains(contains)));
     }
 
+    protected void checkLogDoesNotContain(String route, String contains, int waitForSeconds) {
+        Assertions.assertThatNoException().isThrownBy(() -> Awaitility.await()
+                .atMost(waitForSeconds, TimeUnit.SECONDS)
+                .untilAsserted(() -> Assertions.assertThat(getLogs(route))
+                        .as("log contains " + contains)
```

Comment:
maybe the description is `log does not contain ...`

Reply:
Changed, thanks